templates/terraform/examples/region_network_endpoint_group_appengine.tf.erb

```diff
@@ -0,0 +1,69 @@
+// App Engine Example
+resource "google_compute_global_network_endpoint_group" "<%= ctx[:primary_resource_id] %>" {
+  name                  = "gae-neg"
+  network_endpoint_type = "SERVERLESS"
+  region = "us-central1"

```

Comment:
nit: please align `=`s. it's nice to adhere as best as possible to https://www.terraform.io/docs/configuration/style.html